var/spack/repos/builtin/packages/libceed/package.py

```diff
@@ -55,20 +56,20 @@ def common_make_opts(self):
         if '@:0.2' in spec:
             makeopts += ['NDEBUG=%s' % ('' if '+debug' in spec else '1')]
 
-        elif '@0.4:' in spec:
+        elif '@0.4:' in spec or '@0.5' in spec or '@develop' in spec:
```

Comment:
@jedbrown the `:` in `@0.4:` means that any version at or after `0.4` should trigger this check (so you shouldn't have to modify it).

Reply:
Thanks; amended.